src/script/team/TeamRepository.ts

```diff
@@ -383,18 +383,14 @@ export class TeamRepository extends TypedEventEmitter<Events> {
     }
   };
 
-  private readonly onFeatureConfigUpdate = async (
-    event: TeamFeatureConfigurationUpdateEvent,
-    source: EventSource,
-  ): Promise<void> => {
+  private readonly onFeatureConfigUpdate = async (source: EventSource): Promise<void> => {
     if (source !== EventSource.WEBSOCKET) {
       // Ignore notification stream events
       return;
     }
```

Comment:
I think we can remove this piece of code now. No config update is going to come from the WS anymore 😢

Reply:
I don't think we need the handler at all now, removed it https://github.com/wireapp/wire-webapp/pull/16675/commits/c1570a44b15045831bf5ba06fc10d477cc6100f0 ;)